locales/es-ES/messages.po

```diff
@@ -639,6 +720,12 @@ msgstr ""
 msgid "If given, the pattern mask specifies a regular expression that the candidate's response must match in order to be considered valid"
 msgstr "ya existe par idéntico"
 
+msgid "If the shuffle attribute is true then the delivery engine will randomize the order in which the choices are initially presented. However each choice may be \"shuffled\" of \"fixed\" individually."
+msgstr ""
```

Comment:
- "Selecciones aleatorias" as in https://github.com/oat-sa/extension-tao-itemqti/blob/ec1e8fc53339d25efa911487d5be0297e6c074c8/locales/es-ES/messages.po#L1512
- "Motor de entrega" as in https://github.com/oat-sa/extension-tao-itemqti/blob/ec1e8fc53339d25efa911487d5be0297e6c074c8/locales/es-ES/messages.po#L769

```suggestion
msgid "If the shuffle attribute is true then the delivery engine will randomize the order in which the choices are initially presented. However each choice may be \"shuffled\" of \"fixed\" individually."
msgstr "Si el atributo de selecciones aleatorias está activo, el motor de entrega presentará inicialmente las opciones en orden aleatorio. De todas formas, el orden de cada opción puede establecerse como \"aleatorio\" o \"fijo\" de manera individual."
```